### PR TITLE
Tools: fixed env install for ubuntu 22.04

### PR DESCRIPTION
**Pull request by tridge**
changing to ln -sf is to allow the script to be run twice (if first run fails)

### CODE REVIEW

File: Tools/environment_install/install-prereqs-ubuntu.sh [hunk 1]
```diff
@@ -113,7 +118,7 @@ if [ "$ARM_PKG_CONFIG_NOT_PRESENT" -eq 1 ]; then
     $APT_GET install pkg-config
     if [ -f /usr/share/pkg-config-crosswrapper ]; then
         # We are on non-Ubuntu so simulate effect of installing pkg-config-arm-linux-gnueabihf.
```
Comment (peterbarker) [inline Tools/environment_install/install-prereqs-ubuntu.sh:120]: > changing to ln -sf is to allow the script to be run twice (if first run fails)

The comment right above this change says we are on non-Ubuntu, so *something* is wrong here.

If the comment is right we shouldn't be in here when installing jammy

If the comment is wrong we should remove it.

My guess is the former.

**Review by peterbarker — CHANGES_REQUESTED** [review]

File: Tools/environment_install/install-prereqs-ubuntu.sh (hunk 1 above)
Comment (khancyr) [reply Tools/environment_install/install-prereqs-ubuntu.sh:120]: Or Ubuntu change things to be closer to debian again...

**Review by khancyr — APPROVED** [review]
Beside Peter comment that would need a little investigation it seems good

**Review by peterbarker — APPROVED** [review]
I've tested using Vagrant locally and this works.

The patch I drew issue with seems to do No Harm, at least :-)

**Pull request merged**